README.md

```diff
@@ -1,65 +1,149 @@
-5. Fix linter errors.
-6. **IMPORTANT NOTE**: feel free to research [auto-correct options for Eslint](https://eslint.org/docs/latest/user-guide/command-line-interface#fixing-problems) if you get a flood of errors but keep in mind that correcting style errors manually will help you to make a habit of writing a clean code!
+## 🚀 Live Demo <a name="live-demo"></a>
 
-### Stylelint
+- [Fun zone](https://deploy-preview-8--funnyzone.netlify.app/)
 
-1. Run
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-   ```
-   npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
-   ```
+## 💻 Getting Started <a name="getting-started"></a>
 
-   *not sure how to use npm? Read [this](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).*
+To get a local copy up and running, follow these steps.
 
-2. Copy [.stylelintrc.json](./.stylelintrc.json) to the root directory of your project.
-3. **Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.**
-   - If you think that change is necessary - open a [Pull Request in this repository](../README.md#contributing) and let your code reviewer know about it.
-4. Run `npx stylelint "**/*.{css,scss}"` on the root of your directory of your project.
-5. Fix linter errors.
-6. **IMPORTANT NOTE**: feel free to research [auto-correct options for Stylelint](https://stylelint.io/user-guide/usage/options) if you get a flood of errors but keep in mind that correcting style errors manually will help you to make a habit of writing a clean code!
+### Prerequisites
 
-## Test/Deployment Actions
+In order to run this project you need:
 
-Feel free to add your own deployment actions which can run your tests and deploy to Heroku.
+- Nodejs
+- A text editor of your choice
+- A properly working pc or laptop
 
-Make sure that you do not modify the [`.github/workflows/linters.yml`](.github/workflows/linters.yml) but that you create a separe GitHub Action workflow file for that.
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/naolkidanu/game-web-app.git
+```
+
+### Install
+
+Install this project with node packages:
+
+```sh
+  cd my-project
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npm start
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  npm jest
+```
+
+### Deployment
+
+You can deploy this project using: GitHub but if you want to deploy make sure you let me know first :3
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Naol Kidanu Merdasa**
+
+- GitHub: [Naol Kidanu Merdasa](https://github.com/naolkidanu/)
+- Twitter: [Naol Kidanu Merdasa](https://twitter.com/Naolkidanu)
```

Comment:
- [ ] As indicated by the previous reviewer, Twitter (X) has it that your username does not exist. Kindly check the characters and correct them or create a new account. Socials are ways of networking and accessing opportunities. :wink: